test/test_sparse_csr.py

```diff
@@ -1503,7 +1503,9 @@ def run_test_block_addmm_addmv(self,
     @skipCPUIfNoMklSparse
     @unittest.skipIf(not TEST_SCIPY, "SciPy not found")
     @dtypes(*floating_and_complex_types())
-    @dtypesIfCUDA(*floating_and_complex_types_and(torch.half, torch.bfloat16))
+    @dtypesIfCUDA(*floating_and_complex_types_and(
+                  *[torch.half] if SM53OrLater and TEST_CUSPARSE_GENERIC else [],
+                  *[torch.bfloat16] if SM80OrLater and TEST_CUSPARSE_GENERIC else []))
```

Comment:
I don't think `TEST_CUSPARSE_GENERIC` is important here.

Reply:
I am using `mm`. Is it not relevant there as well?

Reply:
You are using dense `mm` though right? The CuSparse Generic API has no BSR support, it is used for `csr x strided` and `csr x csr`